canonical-web-and-design/jaas.ai
# Redirect charm and bundle links to charmhub

## Done
Redirect charm and bundle links to charmhub
Update the blog and discourse external links

## QA
- Open the demo
- Go to /mysql and see it navigates to charmhub
- Check the bundle links do too

## Details
Fixes https://github.com/canonical-web-and-design/app-tribe/issues/511
